pallets/utility/src/lib.rs

```diff
@@ -25,48 +25,43 @@
 //! # Utility Module
 //! A module with helpers for dispatch management.
 //!
-//! - [`utility::Trait`](./trait.Trait.html)
-//! - [`Call`](./enum.Call.html)
-//!
 //! ## Overview
-//!
-//! This module contains three basic pieces of functionality, two of which are stateless:
-//! - Batch dispatch: A stateless operation, allowing any origin to execute multiple calls in a
+//! This module contains the following functionality:
+//! - [`Batch dispatch`]\: A stateless operation, allowing any origin to execute multiple calls in a
 //!   single dispatch. This can be useful to amalgamate proposals, combining `set_code` with
 //!   corresponding `set_storage`s, for efficient multiple payouts with just a single signature
-//!   verify, or in combination with one of the other two dispatch functionality.
-//! - Pseudonymal dispatch: A stateless operation, allowing a signed origin to execute a call from
-//!   an alternative signed origin. Each account has 2**16 possible "pseudonyms" (alternative
-//!   account IDs) and these can be stacked. This can be useful as a key management tool, where you
-//!   need multiple distinct accounts (e.g. as controllers for many staking accounts), but where
-//!   it's perfectly fine to have each of them controlled by the same underlying keypair.
-//!
-//! ## Interface
-//!
-//! ### Dispatchable Functions
+//!   verify, or in combination with one of the other dispatch functionality.
+//! - [`Relayed dispatch`]\: A stateful operation, allowing a signed origin to execute calls on
+//!   behalf of another account. This is useful when a transaction's fee needs to be paid by a third party.
+//!   Relaying dispatch requires the dispatched call to be unique as to avoid replay attacks.
 //!
-//! #### For batch dispatch
-//! * `batch` - Dispatch multiple calls from the sender's origin.
-//!
-//! [`Call`]: ./enum.Call.html
-//! [`Trait`]: ./trait.Trait.html
+//! [`Batch dispatch`]: struct.Module.html#method.batch
+//! [`Relayed dispatch`]: struct.Module.html#method.relay_tx
 
 // Ensure we're `no_std` when compiling for Wasm.
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use codec::{Decode, Encode};
 use frame_support::{
     decl_error, decl_event, decl_module, decl_storage,
     dispatch::PostDispatchInfo,
+    ensure,
     traits::UnfilteredDispatchable,
     weights::{DispatchClass, GetDispatchInfo, Weight},
     Parameter,
 };
-use frame_system::{self as system, ensure_root};
-use sp_runtime::{traits::Dispatchable, DispatchError, DispatchResult};
+use frame_system as system;
+use frame_system::{ensure_root, ensure_signed, RawOrigin};
+use polymesh_common_utilities::{
+    balances::CheckCdd, identity::AuthorizationNonce, identity::Trait as IdentityModule,
```

Comment:
Perhaps `as IdentityTrait` in order for this not to be confused with `identity_pallet::Module`.